app/Actions/Importer/RfcImporter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Actions\Importer;
+
+use App\Models\PendingRfc;
+use Illuminate\Console\Command;
+
+class RfcImporter
+{
+    public function __invoke(Command $command, array $rfcs = []): void
```

Comment:
Passing commands to write to the console output isn't the best idea. It makes testing a lot harder. Instead you could emit events from within this importer, and handle them from within the command class.